### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Kerry-zzx** [taoxugit/AttnGAN#87]
block = nn.Sequential(
        nn.Upsample(scale_factor=2, mode='nearest'),
        conv3x3(in_planes, out_planes * 2),
        nn.BatchNorm2d(out_planes * 2),
        GLU())

Why out_planes * 2? This will cause the problem of dimension.
Input is : self.upsample1 = upBlock(ngf, ngf // 2)
The channels don't change!!!
